qiskit/extensions/quantum_initializer/isometry.py

```diff
@@ -93,17 +93,19 @@ def __init__(self, isometry, num_ancillas_zero, num_ancillas_dirty):
 
         num_qubits = int(n) + num_ancillas_zero + num_ancillas_dirty
 
-        super().__init__("isometry", num_qubits, 0, [isometry])
+        super().__init__("isometry", num_qubits, [isometry])
 
     def _define(self):
         # TODO The inverse().inverse() is because there is code to uncompute (_gates_to_uncompute)
         #  an isometry, but not for generating its decomposition. It would be cheaper to do the
         #  later here instead.
-        gate = self.inverse().inverse()
+        gate = self.iso_inverse()
+        gate = gate.inverse()
         q = QuantumRegister(self.num_qubits)
         iso_circuit = QuantumCircuit(q)
         iso_circuit.append(gate, q[:])
         self.definition = iso_circuit
+        # self.params = []
```

Comment:
debug code?

Reply:
The inverse raises a warning because params type is numpy.ndarray. This line suppress the warning. 
I removed the line in the commit 'revision'.